test/e2e/pkg_test.go

```diff
@@ -299,6 +299,9 @@ func TestConfigurationWithDigest(t *testing.T) {
 	)
 }
 
+// TestUpgradeDependencyVersion tests that a dependency version is upgrades when the parent configuration is updated.
+// The packages used in this test are build and pushed manually and the manifests must remain unchanged to ensure
```

Comment:
```suggestion
// The packages used in this test are built and pushed manually and the manifests must remain unchanged to ensure
```
Can you point to the folder with the crossplane.yaml too?